packages/playwright/PKGBUILD

```diff
@@ -0,0 +1,34 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-playwright
+_pkgname=playwright-python
+pkgver=v1.29.0.r4.g2d0df8e
+pkgrel=1
+pkgdesc='Python version of the Playwright testing and automation library.'
+arch=('any')
+url='https://github.com/Microsoft/playwright-python'
+license=('Apache')
+depends=('mypy' 'python' 'python-auditwheel' 'python-autobahn' 'python-black' 'python-flake8' 'python-flaky' 'python-greenlet' 'python-objgraph' 'python-pillow' 'python-pixelmatch' 'python-pre-commit' 'python-pyee' 'python-pyopenssl' 'python-pytest' 'python-pytest-asyncio' 'python-pytest-cov' 'python-pytest-repeat' 'python-pytest-timeout' 'python-pytest-xdist' 'python-requests' 'python-service-identity' 'python-twine' 'python-twisted' 'python-types-pyopenssl')
+makedepends=('python-setuptools' 'python-wheel')
+options=(!emptydirs)
+source=("git+https://github.com/Microsoft/$_pkgname.git")
```

Comment:
use https://pypi.org/project/playwright/ with https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-lib-setuptools and maybe better with https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-lib-PEP517 since there is https://github.com/microsoft/playwright-python/blob/main/pyproject.toml

Reply:
For your first suggestion, I cannot use https://pypi.org/project/playwright/ because there are not source files. The only source files I see are in GitHub.

Can I use https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-lib-PEP517 for all source files that don't have `setup.py` but they have `pyproject.toml` ? It could be a good idea.